lua/lspconfig/util.lua

```diff
@@ -87,8 +87,8 @@ end
 function M.create_module_commands(module_name, commands)
   for command_name, def in pairs(commands) do
     local opts = M._parse_user_command_options(def)
-    api.nvim_create_user_command(command_name, function(...)
-      require('lspconfig')[module_name].commands[command_name][1](...)
+    api.nvim_create_user_command(command_name, function(info)
+      require('lspconfig')[module_name].commands[command_name][1](unpack(info.fargs))
```

Comment:
The old way would only provide `fargs` like so: `require'lspconfig'[%q].commands[%q][1](<f-args>)`. The Lua equivalent of this is simply `unpack(info.fargs)`!